packages/api-server/src/__tests__/api.test.ts

```diff
@@ -38,28 +38,14 @@ afterAll(async () => {
   await fastifyInstance.close()
 })
 
-describe('withFunctions', () => {
-  // Deliberately using `toBe` here to check for referential equality.
-  it('returns the same fastify instance', async () => {
-    expect(returnedFastifyInstance).toBe(fastifyInstance)
-  })
-
-  it('configures the `@fastify/url-data` and `fastify-raw-body` plugins', async () => {
+describe('redwoodFastifyAPI', () => {
+  it.only('configures the `@fastify/url-data` and `fastify-raw-body` plugins', async () => {
     const plugins = fastifyInstance.printPlugins()
 
     expect(plugins.includes('@fastify/url-data')).toEqual(true)
     expect(plugins.includes('fastify-raw-body')).toEqual(true)
   })
 
-  it('configures two additional content type parsers, `application/x-www-form-urlencoded` and `multipart/form-data`', async () => {
-    expect(
-      fastifyInstance.hasContentTypeParser('application/x-www-form-urlencoded')
-    ).toEqual(true)
-    expect(fastifyInstance.hasContentTypeParser('multipart/form-data')).toEqual(
-      true
-    )
-  })
```

Comment:
These content parsers are now encapsulated in the api plugin, they're not in the global scope